modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita.adoc

```diff
@@ -199,7 +193,6 @@ Note: If you are using MySQL, add `?allowMultiQueries=true` to the URL. For exam
     Also, if you are migrating to Bonita 7.9+, you must upgrade your database server to MySQL 8.0, see <<mysql8,Migrating to Bonita 7.9+ using MySQL>> specific procedure below.
```

Comment:
you are migrating to Bonita 7.9+, => you are updating to Bonita 7.9+,

Reply:
Same as above